scss/bootstrap/components/civicase/_notification-badge.scss

```diff
@@ -0,0 +1,52 @@
+.crm_notification-badge {
+  background: $brand-primary;
+  border-radius: $border-radius-base;
+  color: $crm-white;
+  display: inline-block;
+  font-size: $font-size-base;
+  line-height: 14px;
+  margin: 0 2px;
+  padding: 5px 8px;
+  text-align: center;
+  vertical-align: middle;
+  white-space: nowrap;
+
+  &:first-child {
+    margin-left: 0;
+  }
+
+  &:last-child {
+    margin-right: 0;
+  }
+
+  &:hover {
+    color: $crm-white;
+    opacity: 0.8;
+    text-decoration: none;
+  }
+}
+
+.crm_notification-badge__active {
```

Comment:
`active` will have a white background and colored icon and text. I have given screenshot in description area, how it will look like. We can use it when we have active tab.